Lombiq.DataTables/package.json

```diff
@@ -1,15 +1,49 @@
 {
   "private": true,
-  "scripts": {
-    "dotnet-prebuild": "gulp"
+  "// inactive dependencies - currently replaced by static files with custom fixes": {
+    "datatables.net": "1.10.25",
+    "datatables.net-bs4": "3.2.2"
```

Comment:
I'm not sure if this is necessary information. Also weird formatting with the slashes inside the string.

Reply:
It's necessary because this should be fixed in an upcoming issue.

The slashes indicate it's a comment. I've started to use this notation in lack of comment support in `json` and lack of `json5` support in VS.

Reply:
I see. So this is basically a TODO comment. I'd prefer if you opened an issue for it and then deleted the comment from here. Issues are easier to track than comments and once the issue is up the comment is superfluous.

Reply:
Here we are: https://github.com/Lombiq/Orchard-Data-Tables/issues/120.

Reply:
Thanks.